src/Fields/Fields.jl

```diff
@@ -393,5 +393,64 @@ function set!(f::Function, field::Field, args = ())
     return nothing
 end
 
+#=
+This function can be used to truncate the printing
+of ClimaCore `Field` types, which can get rather
+long.
+
+# Example
+```
+import ClimaCore
+ClimaCore.Fields.truncate_printing_field_types() = true
+```
+=#
+truncate_printing_field_types() = false
+
+function Base.show(io::IO, ::Type{T}) where {T <: Fields.Field}
+    if truncate_printing_field_types()
+        print(io, print_truncated_field_type(T))
+    else
+        invoke(show, io, print_truncated_field_type(T))
```

Comment:
shouldn't this be
```suggestion
        invoke(show, Tuple{IO, Type}, io, T)
```

Reply:
I've added a test to make sure the standard printing isn't broken, too.